code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1373,11 +1373,7 @@
 
 /datum/reagent/healium/on_mob_metabolize(mob/living/L)
 	. = ..()
-	ADD_TRAIT(L, TRAIT_KNOCKEDOUT, type)
-
-/datum/reagent/healium/on_mob_end_metabolize(mob/living/L)
-	REMOVE_TRAIT(L, TRAIT_KNOCKEDOUT, type)
-	return ..()
+	L.Sleeping(min(L.AmountSleeping() + rand(100, 150), rand(200, 400)))
```

Comment:
Make a child of the sleeping status effect. You want a perma-sleep one here, until `on_mob_end_metabolize()` is called, not the regular timed sleep.